### PR TITLE
Marcos.cifuentes/setting main activity

### PR DESCRIPTION
**Pull request by MarcosCGlobant**
In this PR I added the Main Activity UI, setting the View Pager with the fragments.

- Added the dependencies needed
- Designed the main activity UI
- Set the View Pager Adapter and initialized the View Pager on the activity
- Created Types fragments and classes

GIF of the UI:

UI design

### CODE REVIEW

**Review by lucianoeli — APPROVED** [review]

File: app/src/main/java/com/globant/pokemontcgapp/view/activity/MainPokemonActivity.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        Glide.with(applicationContext).load(R.drawable.app_logo).into(binding.activityMainAppTitleIcon)
+
+        val listOfFragmentNames: List<String> =
+            listOf(getString(R.string.pokemon_types), getString(R.string.pokemon_supertypes), getString(R.string.pokemon_subtypes))
+        TabLayoutMediator(binding.tabLayout, viewPager) { tab, position -> tab.text = listOfFragmentNames[position] }.attach()
+    }
+
+    companion object {
+        fun getIntent(context: Context?): Intent {
+            return Intent(context, MainPokemonActivity::class.java)
```
Comment (mauroLonghi) [inline app/src/main/java/com/globant/pokemontcgapp/view/activity/MainPokemonActivity.kt]: Pls, change the two littles thinks, 1) change Context? to Context, becouse it's bad the null condition...2) change the return to _:Intent = _

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -4,15 +4,43 @@
...
+        android:background="@drawable/gradient_appbar_background"
+        app:layout_constraintTop_toBottomOf="@id/activity_main_app_title_icon">
+
+        <com.google.android.material.tabs.TabLayout
+            android:id="@+id/tabLayout"
+            style="@style/AppTabLayout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:tabGravity="fill"
+            app:tabIndicatorColor="@color/red"
```
Comment (mauroLonghi) [inline app/src/main/res/layout/activity_main.xml]: Pls use another name more representative

Comment (MarcosCGlobant) [reply app/src/main/res/layout/activity_main.xml]: changed, thanks

File: app/src/main/java/com/globant/pokemontcgapp/view/activity/MainPokemonActivity.kt (hunk 1 above)
Comment (MarcosCGlobant) [reply app/src/main/java/com/globant/pokemontcgapp/view/activity/MainPokemonActivity.kt]: refactored, thanks

**Review by SartiMau — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Pull request merged**